specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -8318,6 +8952,253 @@
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the capacity reservation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CapacityReservationInstanceView"
+        }
+      ],
+      "description": "The instance view of a capacity reservation that includes the name of the capacity reservation. It is used for the response to the instance view of a capacity reservation group."
+    },
+    "CapacityReservationProperties": {
+      "type": "object",
+      "properties": {
+        "reservationId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique id generated and assigned to the capacity reservation by the platform. <br><br> Does not change throughout the lifetime of the capacity reservation."
+        },
+        "virtualMachinesAssociated": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines associated with the capacity reservation."
+        },
+        "provisioningTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The date when the capacity reservation was last time updated."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state, which only appears in the response."
+        },
+        "instanceView": {
+          "$ref": "#/definitions/CapacityReservationInstanceView",
+          "readOnly": true,
+          "description": "The Capacity reservation instance view."
+        }
+      },
+      "description": "Properties of the Capacity reservation."
+    },
+    "CapacityReservation": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CapacityReservationProperties"
+        },
+        "sku": {
+          "$ref": "#/definitions/Sku",
+          "description": "SKU of the resource for which capacity needs be reserved. The SKU name and capacity is required to be set. Currently VM Skus with the capability called 'CapacityReservationSupported' set to true are supported. Refer to List Microsoft.Compute SKUs in a region (https://docs.microsoft.com/rest/api/compute/resourceskus/list) for supported values."
+        },
+        "zones": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Availability Zone to use for this capacity reservation. The zone has to be single value and can be assigned only during creation. If not provided, the reservation supports only non-zonal deployments. If provided, enforces VM/VMSS using this capacity reservation to be in same zone."
```

Comment:
The zone has to be single value and also should be part for the list of zones specified during the capacity reservation group creation. The zone can be assigned only during creation.